frontend/controller/actions/group.js

```diff
@@ -532,17 +615,17 @@ export default (sbp('sbp/selectors/register', {
     // chatroom without any extra steps, and, in particular, it enables joining
     // the #General chatroom upon joining a group, in a single step.
     if ([CHATROOM_PRIVACY_LEVEL.GROUP, CHATROOM_PRIVACY_LEVEL.PUBLIC].includes(params.data.attributes.privacyLevel)) {
-      await sbp('gi.actions/out/shareVolatileKeys', {
-        destinationContractID: params.contractID,
-        destinationContractName: 'gi.contracts/group',
-        contractID: message.contractID(),
+      /* await sbp('gi.actions/out/shareVolatileKeys', {
+        contractID: params.contractID,
+        contractName: 'gi.contracts/group',
+        subjectCntractID: message.contractID(),
         keyIds: '*'
-      })
+      }) */
```

Comment:
@corrideat If this `if` branch does nothing then maybe it can be removed entirely?